.github/workflows/ci.yml

```diff
@@ -168,19 +168,21 @@ jobs:
             type=semver,pattern={{version}}
             type=edge
             type=ref,event=pr
-            type=ref,event=branch,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
+            type=ref,event=branch,sufix=-rc,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
```

Comment:
Double checking, suffix has two F's. Does this need to be "suffix=-rc"?

Reply:
oh yes! good catch 😅